composeApp/src/ooniMain/composeResources/values/strings-organization.xml

```diff
@@ -19,4 +19,11 @@
     <string name="performance_datausage" translatable="false">5 - 200 MB</string>
 
     <string name="Settings_About_Label">About OONI</string>
+    <string name="Settings_About_Content_Paragraph">The Open Observatory of Network Interference (OONI) is a free software project under The Tor Project that aims to increase transparency of internet censorship around the world.\n\nSince 2012, OONI\'s global community has been measuring networks in more than 200 countries. Some of these measurements serve as evidence of internet censorship.</string>
```

Comment:
The backspace in `OONI\'s` is showing up in the app. If we switch to the curly version `’` there's no need to escape it.

Reply:
This is markdown actually. I have to update the view to show the markdown properly.

Reply:
Ah ok, got it. The curly version would work anyway.

Reply:
https://github.com/ooni/probe-multiplatform/pull/81